car/events.py

```diff
@@ -3,6 +3,7 @@
 
 logger = logging.getLogger(__name__)
 
+
 # a simple event framework so we can connect together decoupled logic
 class EventHandler:
```

Comment:
This whole file didn't need to change with this suppress_logs functionality, but I couldn't help myself.
Only some types of events flood the logs, so I moved the flag into the event itself rather than implement log flood prevention in all events.

This refactor means that half the code in this class can move into shared in a future PR, and overall code volume will be reduced. It will also mean that the pit app will get the ability to do log suppression ... something it currently doesn't need.